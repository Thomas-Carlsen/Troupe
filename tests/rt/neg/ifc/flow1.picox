let
   val h = 42 raisedTo {secret}
   fun f () = adv 1
   fun g () = adv 2
in
   if h then spawn(f) else spawn(g) 
end
