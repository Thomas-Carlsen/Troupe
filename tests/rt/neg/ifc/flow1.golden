2019-03-07T12:08:26.594Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: 11544b50-aa44-4dad-8524-4d77ab27a8a4@{secret}%{secret}
[31mRuntime error in thread 11544b50-aa44-4dad-8524-4d77ab27a8a4@{secret}%{secret}[39m
[31m>> Illegal flow in adv function:[39m
[31m |    pc: {secret}[39m
[31m | block: {secret}[39m
[31m | value: 1@{secret}%{secret}[39m
